### DISCUSSION TITLE
フォローボタンが突然現れるので、意図せずユーザーをフォローしそうになる

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#509] [要望・アイデア・不具合]
ユーザーランキング欄に出ているユーザーさんのプロフィールを見てみようと思って、赤枠のあたりにマウスカーソルを持っていくと、

<img width="279" alt="Screenshot 2023-07-30 at 9 31 26" src="https://github.com/increments/qiita-discussions/assets/1148320/4ac35362-941b-423e-a18f-47d088f90fbc">

突然フォローボタンが現れて、プロフィールを開くつもりがフォローボタンを押してしまいました。
（フォローボタンが出てくるとは思わなかったので、「えっ、なんでいきなりフォローしちゃったん！？」と慌ててフォロー解除しました💦）

<img width="281" alt="Screenshot 2023-07-30 at 9 31 29" src="https://github.com/increments/qiita-discussions/assets/1148320/80ad48ef-206d-49fa-911d-cfcdb445f3d6">

このような誤フォローが発生しないようにUIの改善をお願いしたいです。

個人的な感覚からすると、ユーザーランキングからいきなり誰かをフォローしようと思うことは滅多にないので、フォローボタンの表示は不要なんじゃないかなーと思っています。

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご意見ありがとうございます！
ご不便をおかけして申し訳ありません🙏

社内で話し合ったところ、現時点で思いつく対応はフォローボタンを無くすことでした…
（最初からボタンを表示させておくにはスペースが狭いため…）
ですが我々としてはフォローへの導線も残しておきたく…

なので申し訳ないのですが、一旦は現状のまま他にいい方法がないかを検討します。
同様のご意見が多いようであれば優先度を上げて対応したいと思います🙇

**Reply by nkay0** [reply]
IDをホバーするとプロフィールの小窓がポップアップで出てきてその中にフォローボタンをつけるというのはどうでしょう。

**Reply by JunichiIto** [reply]
Twitterみたいな動きですね。たしかにこういう風になってる方がありがたいです。
<img width="436" alt="Screenshot 2023-07-31 at 17 10 59" src="https://github.com/increments/qiita-discussions/assets/1148320/c605220e-00da-4a7d-a641-422b9b77b23f">

**Reply by yoshinoyuya** [reply]
ポップアップ先は上下ではなく右側が良い気がします。
他のユーザーにカーソルを移しづらくなる可能性があると思います。

**Reply by tanaka-asami415** [reply]
@nkay0 @JunichiIto @yoshinoyuya 
ありがとうございます！🙏

社内でも ご提案いただいた内容でよさそう！となっていまして、
直近の開発優先度などから考えると先になってしまうかもしれませんが、頂いた方法で検討したいと思います！

**Comment by wataru86** [comment]
@JunichiIto cc @nkay0 @yoshinoyuya

お待たせしております。
タグ・ユーザーランキングを変更する機会があったので、まずはフォローボタンを外す対応をいたしました。
https://qiita.com/release-notes/2024-04-17

フォローのUI改善については時間がかかるかもしれませんが、引き続き検討していきたいとおもいます。

この度はご意見いただきありがとうございました🙇✨

**Reply by JunichiIto** [reply]
@wataru86 返信が遅くなりましたが、フォローボタンがなくなっていることを確認しました。
ご対応どうもありがとうございました！
